### PR TITLE
Add Incidents API

### PR DESCRIPTION
**Pull request by BenAlfasi**
API review - https://microsoftgraph.visualstudio.com/_git/onboarding?path=%2Freviews%2F10142-m365defender-incidents-api%2Fapi.md&_a=preview

### CODE REVIEW

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: api-reference/beta/resources/m365alertcomment.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "m365AlertComment resource type"
+description: "An analyst generated comment that is associated with an alert or incident. "
```
Comment (JarbasHorst) [inline api-reference/beta/resources/m365alertcomment.md:3]: Please remove the empty space before the last Quotation mark. Suggested revision:

- "An analyst generated comment that is associated with an alert or incident."

File: changelog/Microsoft.M365.Defender.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          "Id": "de26487d-676f-4dd1-b4b3-55d39ee37879",
+          "ApiChange": "EnumType",
+          "ChangedApiName": "m365AlertSeverity",
+          "ChangeType": "Addition",
+          "Description": "Added the **m365AlertSeverity** enumeration type.",
+          "Target": "m365AlertSeverity"
+        },
+        {
+          "Id": "de26487d-676f-4dd1-b4b3-55d39ee37879",
+          "ApiChange": "ComplexType",
```
Comment (JarbasHorst) [inline changelog/Microsoft.M365.Defender.json:39]: ComplexType is not on the list of approved values for ApiChange. Please change this to "Resource".

File: changelog/Microsoft.M365.Defender.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+          "ChangedApiName": "m365AlertComment",
+          "ChangeType": "Addition",
+          "Description": "Added the [m365AlertComment](https://docs.microsoft.com/en-us/graph/api/resources/m365AlertComment?view=graph-rest-beta) resource type.",
+          "Target": "m365AlertComment"
+        },
+        {
+          "Id": "de26487d-676f-4dd1-b4b3-55d39ee37879",
+          "ApiChange": "EntityType",
```
Comment (JarbasHorst) [inline changelog/Microsoft.M365.Defender.json:47]: EntityType is not on the list of approved values for ApiChange. Please change this to "Resource".

File: changelog/Microsoft.M365.Defender.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+          "ChangedApiName": "incident",
+          "ChangeType": "Addition",
+          "Description": "Added the [incident](https://docs.microsoft.com/en-us/graph/api/resources/incident?view=graph-rest-beta) resource",
```
Comment (JarbasHorst) [inline changelog/Microsoft.M365.Defender.json:50]: For clarity and consistency, please add type after resource. Suggested revision:

- ...urces/incident?view=graph-rest-beta) resource type.

File: changelog/Microsoft.M365.Defender.json [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+          "Target": "incident"
+        },
+        {
+          "Id": "de26487d-676f-4dd1-b4b3-55d39ee37879",
```
Comment (JarbasHorst) [inline changelog/Microsoft.M365.Defender.json:54]: I don't see this resource type in your PR. Please remove this entry from this changelog record.

File: changelog/Microsoft.M365.Defender.json [hunk 6]
```diff
@@ -0,0 +1,70 @@
+{
+  "changelog": [
+    {
+      "ChangeList": [
```
Comment (JarbasHorst) [inline changelog/Microsoft.M365.Defender.json:4]: @BenAlfasi your PR introduces two new methods (list incidents and get incident) that aren't listed in the changelog record provided. Please, include these changes also to the changelog.

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: api-reference/beta/resources/incident.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
+---
+title: "incident resource type"
+description: "An incident in Microsoft 365 Defender is a collection of correlated alerts and associated metadata that reflect the story of an attack."
+author: "bealfasi"
```
Comment (JarbasHorst) [inline api-reference/beta/resources/incident.md:4]: Please, replace bealfasi with BenAlfasi

File: api-reference/beta/resources/m365alertcomment.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+author: "bealfasi"
```
Comment (JarbasHorst) [inline api-reference/beta/resources/m365alertcomment.md:4]: Please, replace bealfasi with BenAlfasi

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: api-reference/beta/resources/enums.md [hunk 9]
```diff
@@ -2486,3 +2486,54 @@ Possible values for user account types (group membership), per Windows definitio
...
+
+### m365AlertDetermination values 
+
+| Member                     | Description                                                                                                                  |
+| :--------------------------| :--------------------------------------------------------------------------------------------------------------------------- |
+| unknown		             | No determination value was set yet.                                                                                          |
+| apt			             | A true positive alert that detected an advanced persistent threat.                                                           |
+| malware                    | A true positive alert that detected malicious software.                                                                      |
+| securityPersonnel          | A true positive alert that detected valid suspicious activity that was performed by someone on the customer's security team. |
+| securityTesting            | The alert detected valid suspicious activity that was performed as part of known security testing.                           |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/enums.md:2517]: Articles help readers and translation software identify the nouns and modifiers in a sentence. Please add a before known. Suggested revision:

- ...as part of a known security testing.

File: api-reference/beta/resources/enums.md [hunk 10]
```diff
@@ -2486,3 +2486,54 @@ Possible values for user account types (group membership), per Windows definitio
...
+| multiStagedAttack          | A true positive alert that detected multiple kill-chain attack stages.                                                       |
+| compromisedUser            | A true positive alert that detected that the intended user's credentials were compromised or stolen.                         |
+| phishing                   | A true positive alert that detected a phishing email.                                                                        |
+| maliciousUserActivity      | A true positive alert that detected the logged on user tries to perform malicious activities.                                |
+| clean                      | A false alert, no suspicious activity.                                                                                       |
+| insufficientData           | A false alert, without enough information to prove otherwise.                                                                |
+| confirmedUserActivity      | The alert caught a true suspicious activity, but it’s OK, it’s a known user activity.                                        |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/enums.md:2525]: It's is the contraction meaning it is or it has. For clarity, I recommend you to not use the contraction. Also, for readability, I recommend you to rephrase the sentence:

- The alert caught a true suspicious activity that is considered OK because it is a known user activity.

File: api-reference/beta/resources/enums.md [hunk 11]
```diff
@@ -2486,3 +2486,54 @@ Possible values for user account types (group membership), per Windows definitio
...
+| lineOfBusinessApplication  | The alert caught a true suspicious activity, but it’s OK, it’s a known and confirmed internal application.                   |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/enums.md:2526]: It's is the contraction meaning it is or it has. For clarity, I recommend you to not use the contraction. Also, for readability, I recommend you to rephrase the sentence:

- The alert caught a true suspicious activity that is considered OK because it is a known and confirmed internal application.

File: api-reference/beta/resources/incident.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+doc_type: resourcePageType
+---
+
+# incident resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+An incident in Microsoft 365 Defender is a collection of correlated alerts and associated metadata that reflect the story of an attack.
```
Comment (JarbasHorst) [inline api-reference/beta/resources/incident.md:16]: Please, replace reflect with reflects.

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: api-reference/beta/resources/incident.md [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+|[List incidents](../api/incident-list.md)|[incident](../resources/incident.md) collection|Get a list of the [incident](../resources/incident.md) objects and their properties.|
+|[Get incident](../api/incident-get.md)|[incident](../resources/incident.md)|Read the properties and relationships of an [incident](../resources/incident.md) object.|
+
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|assignedTo|String|Owner of the incident, or null if no owner is assigned. Free editable text.|
+|classification|m365AlertClassification|The specification for the incident. Possible values are: `unknown`, `falsePositive`, `truePositive`, `benignPositive`, `unknownFutureValue`.|
+|comments|[m365AlertComment](../resources/m365alertcomment.md) collection|Array of comments created by SecOps when the incident is managed.|
```
Comment (JarbasHorst) [inline api-reference/beta/resources/incident.md:33]: For clarity, please also spell out SecOps. For clarity, I recommend you to rephrase the sentence. Suggested revision:

- ..created by the Security Operations (SecOps) team when the incident...

File: api-reference/beta/resources/m365alertcomment.md [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+An analyst generated comment that is associated with an alert or incident. 
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|comment|String|The comment text.|
+|createdByDisplayName|String|The person or app name that submitted the comment.|
+|createdDateTime|DateTimeOffset|The time the comment was submitted.|
```
Comment (JarbasHorst) [inline api-reference/beta/resources/m365alertcomment.md:24]: For clarity, please add "when" before "the". Suggested revision:

- The time when the comment...

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review.

File: api-reference/beta/resources/incident.md [hunk 15]
```diff
@@ -0,0 +1,135 @@
...
+|displayName|String|The incident name.|
+|id|String|Unique identifier to represent the incident.|
+|incidentWebUrl|String|URL for the incident page in Microsoft 365 Defender portal.|
+|lastUpdateDateTime|DateTimeOffset|Time when the incident was last updated.|
+|redirectIncidentId|String|Only populated in case an incident is grouped together with another incident, as part of the logic that processes incidents.|
+|severity|m365AlertSeverity|Indicates the possible impact on assets. The higher the severity the bigger the impact. Typically higher severity items require the most immediate attention. Possible values are: `unknown`, `informational`, `low`, `medium`, `high`, `unknownFutureValue`.|
+|status|incidentStatus|The status of the incident. Possible values are: `active`, `resolved`, `redirected`, `unknownFutureValue`.|
+|tags|String collection|Array of custom tags associated with an incident.|
+
+
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/incident.md:45]: Lines 46-95:
@BenAlfasi , please take note that I moved the 4 enums from enums.md to this parent resource topic (incident.md) where these enums are used.
This follows our enum guidance for option 2: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103158/Enumerations?anchor=second-option%3A-in-parent-resource-topic

File: api-reference/beta/resources/incident.md [hunk 16]
```diff
@@ -0,0 +1,135 @@
...
+---
+
+# incident resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+An incident in Microsoft 365 Defender is a collection of correlated alerts and associated metadata that reflects the story of an attack.
+
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/incident.md:18]: Line 17 and GLOBAL:
@BenAlfasi , as your related api.md file pointed out, there's a set of similar REST API docs at https://docs.microsoft.com/en-us/microsoft-365/security/defender/api-incident?view=o365-worldwide.

Taking the incident resource as an example, does the extra content in the corresponding topic for this resource (copied below) apply here as well? If so, can you include some richer information please? You can have that info just starting in a new paragraph after line 18, and before the "Methods" section in line 21.

"Events from different entities in your organization are automatically aggregated by Microsoft 365 Defender. You can use the incidents API to programatically access your organization's incidents and related alerts."

File: api-reference/beta/api/incident-list.md [hunk 17]
```diff
@@ -0,0 +1,115 @@
...
+
+## Optional query parameters
+
+
+This method supports the following [OData query parameters](/graph/query-parameters) to help customize the response:
+
+- `$select`
+- `$skip`
+- `$top`
+- `$filter`
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/incident-list.md:47]: Line 47:
@BenAlfasi , for the $filter query parameter, do you know which properties support it? And which operations?

FYI many customers look for this kind of query parameter support details.

If you or your engineering team can get those details, you can please add that info here. That would be similar to how it's documented in https://docs.microsoft.com/en-us/microsoft-365/security/defender/api-list-incidents?view=o365-worldwide, also copied below:

"$filter on the lastUpdateTime, createdTime, status, and assignedTo properties"

File: api-reference/beta/api/incident-list.md [hunk 18]
```diff
@@ -0,0 +1,115 @@
...
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /security/incidents
+GET /security/incidents?$top=10
+GET /security/incidents?$filter={property}+eq+'{property-value}'
+```
+
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/incident-list.md:38]: Lines 35-36:
Please move these lines and describe them in the next section "Optional query parameters".

You can see an example of how to document this in https://docs.microsoft.com/en-us/graph/api/attacksimulationroot-list-simulations?view=graph-rest-beta&tabs=http#optional-query-parameters

File: api-reference/beta/api/incident-list.md [hunk 19]
```diff
@@ -0,0 +1,115 @@
...
+}
+-->
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "value": [
+    {
+      "@odata.type": "#microsoft.graph.incident",
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/incident-list.md:91]: Lines 93-108:
@BenAlfasi , can you provide some real-looking data in the example response? That would be very helpful for customers.

File: api-reference/beta/api/incident-get.md [hunk 20]
```diff
@@ -0,0 +1,100 @@
...
+    "determination": "String",
+    "status": "String",
+    "severity": "String",
+    "tags": [
+      "String"
+    ],
+    "comments": [
+      {
+        "@odata.type": "microsoft.graph.m365AlertComment"
+      }
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/incident-get.md:95]: Lines 79-94:
@BenAlfasi , can you provide some real-looking data in the example response? That would be very helpful for customers.

File: api-reference/beta/api/incident-list.md [hunk 21]
```diff
@@ -0,0 +1,115 @@
...
+doc_type: apiPageType
+---
+
+# List incidents
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Get a list of the [incident](../resources/incident.md) objects and their properties.
+
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/incident-list.md:17]: Line 16 and GLOBAL:
@BenAlfasi , can this operation topic share some of the rich info in the docs published for your other endpoint: https://docs.microsoft.com/en-us/microsoft-365/security/defender/api-list-incidents?view=o365-worldwide 

The same question goes for other reference topics in this PR.

Using this list operation topic as an example, in the other set of docs, the [list operation topic](https://docs.microsoft.com/en-us/microsoft-365/security/defender/api-list-incidents?view=o365-worldwide) says:

"The list incidents API allows you to sort through incidents to create an informed cybersecurity response. It exposes a collection of incidents that were flagged in your network, within the time range you specified in your environment retention policy. The most recent incidents are displayed at the top of the list. Each incident contains an array of related alerts, and their related entities."

If any of this info applies to the Microsoft Graph endpoint too, please add that to this topic as well. 
Even though in general, we don't like to copy and paste, we would still want the content here to have enough details to help Graph customers.

File: api-reference/beta/api/incident-get.md [hunk 22]
```diff
@@ -0,0 +1,100 @@
...
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+|Permission type|Permissions (from least to most privileged)|
+|:---|:---|
+|Delegated (work or school account)|Incidents.Read.All, Incidents.ReadWrite.All|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|Incidents.Read.All, Incidents.ReadWrite.All|
+
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/incident-get.md:26]: Permissions:
@BenAlfasi , please add these new Incidents.* permissions to this topic as well: https://docs.microsoft.com/en-us/graph/permissions-reference
You can take a look at how the permissions of other workloads are documented there, and provide the same level of details please.

File: api-reference/beta/resources/incident.md [hunk 23]
```diff
@@ -0,0 +1,135 @@
+---
+title: "incident resource type"
+description: "An incident in Microsoft 365 Defender is a collection of correlated alerts and associated metadata that reflects the story of an attack."
+ms.date: 09/09/2021
+author: "BenAlfasi"
+ms.localizationpriority: medium
+ms.prod: "security"
+doc_type: resourcePageType
+---
+
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/incident.md:10]: This comment is about providing more breadth and depth for your customers: is there more background info you can provide about incidents? What's are "incidents", how do you intend them to be used, why would customers want to use them?

There are 3 places you can insert some additional info. 
- First, for the value proposition of incidents (do not delve into the REST API here), you can add that info to the security service-level conceptual overview (https://docs.microsoft.com/en-us/graph/security-concept-overview).
- For API use cases, you can add that info to the API-level overview: https://docs.microsoft.com/en-us/graph/api/resources/security-api-overview?view=graph-rest-beta
- Third is this incident resource topic. This is the main API resource, many customers remotely familiar with the incidents API would likely search for "incident" and get to this incident resource type topic. Please see my next comment for what you can consider adding.

As your PR stands right now, apart from the actual API documentation, this PR doesn't provide much info for the "what" and "why".

File: api-reference/beta/api/incident-list.md [hunk 24]
```diff
@@ -0,0 +1,115 @@
+---
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/incident-list.md:1]: @BenAlfasi , please rename this file as security-list-incidents.md.

Please also fix all the links pointing to this topic to use the appropriate filename.

The filenaming guidance is here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103161/File-naming

File: api-reference/beta/resources/incident.md (hunk 15 above)
Comment (angelgolfer-ms) [reply api-reference/beta/resources/incident.md:45]: Lines 32, 35, 41, and 42:
Action item for @BenAlfasi - for each of the 4 properties that have one of these enums as the type, please provide a link to the corresponding H3 sections that document that enum type.

That follows the enum guidance for option 2, step 3.

File: changelog/Microsoft.M365.Defender.json [hunk 25]
```diff
@@ -0,0 +1,78 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the [incident](https://docs.microsoft.com/en-us/graph/api/resources/incident?view=graph-rest-beta) resource type.",
+          "Target": "incident"
+        },
+        {
+          "Id": "de26487d-676f-4dd1-b4b3-55d39ee37879",
+          "ApiChange": "Method",
+          "ChangedApiName": "incident-get",
+          "ChangeType": "Addition",
+          "Description": "Added the [incident-get](https://docs.microsoft.com/en-us/graph/api/incident-get?view=graph-rest-beta) method on the [incident](https://docs.microsoft.com/en-us/graph/api/resources/incident?view=graph-rest-beta) resource type.",
```
Comment (angelgolfer-ms) [inline changelog/Microsoft.M365.Defender.json:58]: Line 58 and 66:
@BenAlfasi , can you rephrase these descriptions as:
"Added the GET operation for the incident resource type"
"Added the list operation for the incident resource type."

And keep the links to the corresponding operation topic and the incident resource topic.

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
Hi @BenAlfasi , I added a few more comments, please take a look and incorporate them. If you have questions, please ping me on Teams.

File: api-reference/beta/api/incident-get.md (hunk 22 above)
Comment (BenAlfasi) [reply api-reference/beta/api/incident-get.md:26]: @angelgolfer-ms , I have a different PR for the Permissions - https://github.com/microsoftgraph/microsoft-graph-docs/pull/14236 
Should i merge the PRs?

File: api-reference/beta/api/security-get-incident.md [hunk 26]
```diff
@@ -73,28 +73,28 @@ HTTP/1.1 200 OK
...
-    "determination": "String",
-    "status": "String",
-    "severity": "String",
+    "id": 2972395,
+    "incidentWebUrl": "https://security.microsoft.com/incidents/2972395?tid=12f988bf-16f1-11af-11ab-1d7cd011db47",
+    "redirectIncidentId": null,
+    "displayName": "Multi-stage incident involving Initial access & Command and control on multiple endpoints reported by multiple sources",
+    "createdDateTime": "2021-08-13T08:43:35.5533333Z",
+    "lastUpdateDateTime": "2021-09-30T09:35:45.1133333Z",
+    "assignedTo": user@contoso.onmicrosoft.com,
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/security-get-incident.md:83]: Line 83: 
FYI the value which is a string should be enclosed by double quotes. This syntax issue resulted in an API Doctor error. I'm fixing it on your behalf.

File: api-reference/beta/api/security-list-incidents.md [hunk 27]
```diff
@@ -88,26 +98,28 @@ Content-Type: application/json
...
-      ]
+        "@odata.type": "#microsoft.graph.incident",
+        "id": 2972395,
+        "incidentWebUrl": "https://security.microsoft.com/incidents/2972395?tid=12f988bf-16f1-11af-11ab-1d7cd011db47",
+        "redirectIncidentId": null,
+        "displayName": "Multi-stage incident involving Initial access & Command and control on multiple endpoints reported by multiple sources",
+        "createdDateTime": "2021-08-13T08:43:35.5533333Z",
+        "lastUpdateDateTime": "2021-09-30T09:35:45.1133333Z",
+        "assignedTo": user@contoso.onmicrosoft.com,
+        "classification": "TruePositive",
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/security-list-incidents.md:109]: Line 108:
FYI the value which is a string should be enclosed by double quotes. This syntax issue resulted in an API Doctor error. I'm fixing it on your behalf.

File: api-reference/beta/api/security-list-incidents.md [hunk 28]
```diff
@@ -88,26 +98,28 @@ Content-Type: application/json
...
+        "determination": "MultiStagedAttack",
+        "status": "Active",
+        "severity": "Medium",
+        "tags": [
+          "Demo"
+        ],
+        "comments": [
+          {
+		    "comment": "Demo incident",
+		    "createdBy": "alfasi@wcdtestprd.onmicrosoft.com",
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/security-list-incidents.md:119]: Line 119 and GLOBAL:
FYI in examples and other docs that show "customer data", we can show only specific CELA approved fictitious values. Details here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103160/Fictitious-names

File: api-reference/beta/api/security-get-incident.md [hunk 29]
```diff
@@ -73,28 +73,28 @@ HTTP/1.1 200 OK
...
     "tags": [
-      "String"
+      "Demo"
     ],
     "comments": [
       {
-        "@odata.type": "microsoft.graph.m365AlertComment"
+		"comment": "Demo incident",
+		"createdBy": "alfasi@wcdtestprd.onmicrosoft.com",
+		"createdTime": "2021-09-30T12:07:37.2756993Z"
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/security-get-incident.md:95]: Line 94:
FYI we can show only CELA-approved people's names, email addresses, web domains, etc.
Details here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103160/Fictitious-names

File: api-reference/beta/api/security-get-incident.md [hunk 30]
```diff
@@ -73,28 +73,28 @@ HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/security-get-incident.md:75]: Filename:
The GET operation file actually had the correct filename incident-get.md.
The filenaming guidelines are here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103161/File-naming

In the interest of time and to reduce churn, I fixed the filename and links to it in your PR.

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
@BenAlfasi , thanks for incorporating the feedback! 
There were some minor rough edges that I subsequently fixed in the interest of time. Please take a look at my comments for your info.
I answered your question regarding defining the new permissions in PR #14236 ; please take a look.
Other than that permissions definition issue, I think this PR is good to be published actually.
I'll tag this "content review complete" once you take action on PR 14236.

File: api-reference/beta/api/incident-get.md (hunk 22 above)
Comment (BenAlfasi) [reply api-reference/beta/api/incident-get.md:26]: Merged

File: api-reference/beta/api/security-list-incidents.md (hunk 27 above)
Comment (BenAlfasi) [reply api-reference/beta/api/security-list-incidents.md:110]: Thanks!

**Review by angelgolfer-ms — APPROVED** [review]
Hi @BenAlfasi , thanks for adding a section to the security API overview (security-api-overview.md). I gave it a bit more context (e.g., added some links). Please take a look at my last commit.
When you get to a point later on that you can revise the service level conceptual overview (security-conceptual-overview.md) as we talked about on Teams yesterday, then I think you'll extract some of the descriptions from the security-api-overview.md that describes the feature and put that in security-conceptual-overview.md. 
In general, an API overview should be less about the feature behavior, and more about how the related APIs support use cases.

When the permissions are deployed (and API and CSDL are available to public customers), please set the label "ready to merge", and one of us will merge and publish this PR.
